docker-compose-swarm.yml

```diff
@@ -11,15 +12,19 @@ consul:
     - 8500:8500
     restart: always
 
+# Run the application on AWS/GCE as follows:
+# sed -r "s/consul:<CONSUL_IP>/consul:$(shell cat .aws/consul)/" ../docker-compose-swarm.yml > /tmp/swarm.yml
+# COMPOSE=/tmp/swarm.yml PREFIX=cb4 VERSION=4 ../start.bash
 
 # Manually bootstrap the first instance, then...
 # Scale this tier and each additional container/instance will automatically
 # self-configure as a member of the cluster
 couchbase4:
     image: 0x74696d/triton-couchbase:4.0.0
+    net: host
     extra_hosts:
-    - consul:${CONSUL_IP}
-    mem_limit: 128g
+    - "consul:<CONSUL_IP>"
```

Comment:
Right, I saw that later on and should have noted it.

I'm begrudgingly thinking that's the best approach for a problem I've got.